### ISSUE TITLE
Button appears clicked after clicking it

### ISSUE THREAD

**Issue by FourierTransformer** [milligram/milligram#145]
### Version info

**Milligram:** v1.2.4 and 1.3.0
Chrome 55 on Mac OS X and Firefox 51 on Mac OS X and Windows 10

### Test case

https://milligram.github.io/buttons.html

### Steps to reproduce

Click on the button, button color changes to grey, and it stays grey after moving mouse away.

In Chrome, this affects the default, outlined, and clear button. On firefox, only the default button has this issue.

### Expected behavior

The color goes back to purple when clicking/hovering away.

### Actual behavior

The color remains grey after clicking on it.

**Comment by cjpatoilo** [comment]
@FourierTransformer Good call. Also I noticed that Safari 10.0.2 does not support the active button in any of the examples: default, outlined and clear. =/

Thank you for reporting this issue. I will analyze and see what we can do to improve this.

**Comment by jgillich** [comment]
Button has `background-color: $color-secondary` set for `focus`,applying it only to `hover` should fix this (but I don't even use Miligram yet so not touching the code for now :).

**Comment by tommyip** [comment]
So will this be fixed?

**Comment by briancodes** [comment]
Is this project still being actively supported?

**Comment by cjpatoilo** [comment]
@FourierTransformer @tommyip @jgillich I haven't yet been updated if this issue has been resolved in Safari. I notice that other CSS frameworks like [Bootstrap](https://github.com/twbs/bootstrap/blob/v4-dev/dist/css/bootstrap.css#L242-L245), [Foundation](https://github.com/zurb/foundation-sites/blob/develop/dist/css/foundation.css#L2655-L2657) uses style for this behavior on button. For these reasons I'm not sure we should remove this behavior.

**Comment by cjpatoilo** [comment]
@FourierTransformer @tommyip @jgillich 

In other CSS frameworks like [Bootstrap](https://getbootstrap.com/docs/4.5/components/buttons/), [Foundation](https://get.foundation/sites/docs/button.html) defines the style for this button behavior.
For this reason, I believe that we shouldn't remove this behavior. Maybe change this behavior.

Can you say to me any idea better?
For example, in Bootstrap changed the `outline` color.

**Comment by cjpatoilo** [comment]
@FourierTransformer @tommyip @jgillich 

it's not a bug. It's is the normal button behavior. In other CSS frameworks like [Bootstrap](https://getbootstrap.com/docs/4.5/components/buttons/), [Foundation](https://get.foundation/sites/docs/button.html), it is also defined. For this reason, I believe that we shouldn't remove this behavior.

Maybe change this behavior. For example, in Bootstrap changed the `outline` color.
Let's look for some cool ideas to improve this behavior.

**Comment by cjpatoilo** [comment]
@FourierTransformer @tommyip @jgillich 

it's not a bug. It's is the normal button behavior. In other CSS frameworks like [Bootstrap](https://getbootstrap.com/docs/4.5/components/buttons/), [Foundation](https://get.foundation/sites/docs/button.html), it is also defined. For this reason, I believe that we shouldn't remove this behavior.

Maybe change this behavior. For example, in Bootstrap changed the `outline` color.
Let's look for some cool ideas to improve this behavior.

ps: I suggest to change the title of this issue to "Change the appears of the button when active"

**Comment by tommyip** [comment]
I guess the reason why this issue stands out from the other frameworks is that our buttons turn grey on hover/focus, whereas the others turn into a darker shade of the button (which seems to be a common web UX nowadays). Perhaps we should change the color on hover/focus to a different shade of purple (could be different between hover and focus)?

Thanks for looking into this by the way @cjpatoilo.

**Comment by anaraujo** [comment]
Hi! Can I pick this one up? I can change the hover/focus color of the buttons to be a darker shade of the $color-primary :)

**Comment by Andyy-18** [comment]
Hey @FourierTransformer 
Can I work on this issue ?

**Comment by kalyanikakade123** [comment]
I would like to work on this issue. Can you please assign it to me ?

**Comment by SmritiV23** [comment]
I would like to solve this problem, please do assign this task to me.

**Comment by Shanna99** [comment]
Hi can i solve this issue?

**Comment by Hiresh23** [comment]
i would like to solve the problem , i can change the hover colour of button exactly as you need or may be do any other css changes

**Comment by Zaid-Parkar** [comment]
Hello Sir,
I would like to contribute to this project,
I am new to hacktoctoberfest and this would be a great beginning.
Thank you...

**Comment by PrakarshSingh5** [comment]
plz assign this to me

**Comment by APURVAMOGAL** [comment]
CAN YOU EXPLAIN IT IN DETAILS AND WHAT YOU EXPECT...

**Comment by Anshikaa-agarwal** [comment]
Hello sir! I am interested in resolving this issue. May you please assign this issue to me?

**Comment by uzair-ahmed-01** [comment]
ill solve this issue its very easy and i want to contribute to this open source project can you please assign this to me. TQ

**Comment by AtharvSabde** [comment]
i have worked on hover effect animation project.. can you please assign me this

**Comment by Ankitapanda25** [comment]
Could you please assign this to me, I have worked on a lot of projects similar to these and it is easier to fix!

**Comment by ghost** [comment]
hey @cjpatoilo can you please assign this task to me? I can do this for you!
or Is this issue already done?
